### PR TITLE
Update thumb

### PR DESCRIPTION
**Pull request by Lobi29**
<h1 align="center">ISSUE #279</h1>

Untitled drawing (2)

### features added

- Utilize Tailwind CSS to style the scrollbar to match our theme.
- Use media queries to display the scrollbar only on medium-sized screens or larger.

### CODE REVIEW

File: app/globals.css [hunk 1]
```diff
@@ -136,3 +136,26 @@ html.dark {
...
+    width: 15px;
+  }
+
+  .customScrollbar::-webkit-scrollbar-track {
+    background: var(--background);
+    border-radius: 5px;
+  }
+
+  .customScrollbar::-webkit-scrollbar-thumb {
+    background-color: #6025c0;
```
Comment (rithviknishad) [inline app/globals.css:157]: What if the org. deploying leaderboard has a different primary color? #283

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: app/globals.css (hunk 1 above)
Comment (Lobi29) [reply app/globals.css:157]: okay, I will use the colour theme from the tailwind.config.ts file

**Comment by Lobi29** [comment]
Color theme applied, with emphasis on the primary theme

**Comment by sainak** [comment]
Screenshot 2024-03-01 at 8 19 09 PM
Screenshot 2024-03-01 at 8 19 49 PM

**Pull request closed without merging**